minkindr/include/kindr/minimal/implementation/angle-axis-inl.h

```diff
@@ -64,12 +64,15 @@ AngleAxisTemplate<Scalar>::AngleAxisTemplate(const RotationMatrix& matrix) :
 
 template<typename Scalar>
 AngleAxisTemplate<Scalar>::AngleAxisTemplate(const Vector3& angleAxis) {
-    Scalar norm = angleAxis.norm();
-    CHECK_GT(norm, 1e-8);
-    Vector3 axis = angleAxis;
-    axis = axis / norm;
-
-    C_A_B_ = Implementation(norm, axis);
+    const Scalar angle_rad = angleAxis.norm();
```

Comment:
This parameter should also be called `rotationVector` (more correct and consistent with the new getter).